docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -587,9 +587,9 @@ Rather than apply an effect size associated with the intervention to a particula
 
 **How to determine which wasting state transition rates apply to a given simulant:**
 
-#. Determine if a simulant is covered by treatment coverage with the scenario-specific probability :math:`C`
+#. Determine if a simulant is covered by treatment coverage with the scenario-specific probability :math:`C` (which applies to coverage of MAM and SAM equally at the draw level). Each simulant should have a single propensity value for :math:`C` for both MAM and SAM treatment (equally likely to be covered by SAM treatment as MAM treatment).
 
-#. If simulant is uncovered according to #1: apply untreated transition rates (:math:`r_{SAM,ux}` and :math:`r_{MAM,ux}`). If simulant is covered according to #1: determine if the simulant is "effectively" covered using the wasting state-specific probability :math:`E_{SAM}` or :math:`E_{MAM}` corresponding to the wasting state that they occupy at the current time step.
+#. If simulant is uncovered according to #1: apply untreated transition rates (:math:`r_{SAM,ux}` and :math:`r_{MAM,ux}`). If simulant is covered according to #1: determine if the simulant is "effectively" covered using the wasting state-specific probability :math:`E_{SAM}` or :math:`E_{MAM}` corresponding to the wasting state that they occupy at the current time step. NOTE: distributions of :math:`E_{SAM}` and :math:`E_{MAM}` should not be correlated at the draw level. Simulants should have random propensity values for :math:`E_{SAM}` and :math:`E_{MAM}` such that it is possible for them to be effectively covered for one course of MAM treatment and not effectively covered for the next and vice versa.
```

Comment:
Would a different way to say this be, "the probability an individual is cured is independent of past treatment/non-treatment history"?

Reply:
Are we saying it's independent of the _treatment history_, or independent of whether they've _responded_ to past treatments?

Reply:
The probability that an individual responds to treatment is independent of whether they've responded to past treatments.

After talking it over with Rajan, this point has important implications for model structure. If we keep this assumption, we can use standard vivarium tools. If we assume instead that they have a static propensity for treatment response, we will need a custom model for SAM treatment. I am looking into the literature to see which assumption appears more valid right now. Do either of you have an opinion on this already?

Reply:
From this paper, important predictors of treatment response include 1) diarrhea (we can incorporate this into our vicious cycle model later), 2) oedema (which should be associated with SAM/MAM episode in most cases rather than chronic state), 3) antibiotics given with treatment. Other possible factors include distance from treatment center and vitamin A supplementation, although direct evidence was not available in this review.

Given this, I think we should stick with the independence assumption that allows for the use of standard tools. Do you agree? 

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7339430/pdf/12887_2020_Article_2188.pdf

Reply:
Agree, let's use an independence assumption.  I don't understand how this is more amenable to our standard tools, but I'll keep my eye out for the PR on the python repo and maybe that will make it clearer to me.